specification/network/resource-manager/Microsoft.Network/stable/2018-08-01/azureFirewall.json

```diff
@@ -259,22 +259,18 @@
    "definitions":{  
       "AzureFirewallIPConfigurationPropertiesFormat":{  
          "properties":{  
-			"privateIPAddress": {
-			  "type": "string",
-			  "description": "The Firewall Internal Load Balancer IP to be used as the next hop in User Defined Routes."
-			},
-            "subnet":{  
+            "privateIPAddress": {
+              "type": "string",
+              "description": "The Firewall Internal Load Balancer IP to be used as the next hop in User Defined Routes."
+            },
+            "subnet":{
                "$ref":"./network.json#/definitions/SubResource",
                "description":"Reference of the subnet resource. This resource must be named 'AzureFirewallSubnet'."
             },
-			"internalPublicIpAddress":{  
-               "$ref":"./network.json#/definitions/SubResource",
-               "description":"Reference of the PublicIP resource. This field is a mandatory input."
+            "publicIPAddress": {
```

Comment:
You wrote here `publicIPAddress`, but `publicIpAddress`. JSON is case-sensitive, please be sure to put EXACTLY what the endpoint is returning.